docs/admin-guide/troubleshooting.md

```diff
@@ -1,35 +1,30 @@
 
 ## Incident Roles
 
-|                                |                                          |
-| ------------------------------ | ---------------------------------------- |
-| :worker: Incident Responder(s) | The person who has the hands-on keyboard |
-| :captain: Incident Commander   | Communicating with stakeholders          |
-| :talk: Incident Communicator   | Overseeing the incident                  |
+|                                |                                 |
+| ------------------------------ | ------------------------------- |
+| :worker: Incident Responder(s) | Hands on keyboard               |
+| :captain: Incident Commander   | Communicating with stakeholders |
+| :talk: Incident Communicator   | Overseeing the incident         |
 
+The steps below are applicable for Severity ("**Sev**") 1 & 2 level incidents. See the [Escalations](#escalations) section for more detailed steps with times. See [Severity Classification](#severity-classification) for a severity classification rubric.
 
-
-The steps below are applicable for Sev 1 & 2 incidents, see the Escalations section for more detailed steps with times etc..
-
-<span style="font-size: 28px; color: grey">:1:</span> The first responder automatically becomes the incident commander until it is transferred:      <br/>
+:1: The first responder automatically becomes the incident commander until the role is transferred:
```

Comment:
Re numbering: My general view is that we should avoid HTML in markdown where markdown is sufficient. Personally like the style without the tags, but even so think it's best to keep the MD as clean as possible and style with a stylesheet if necessary. 

Note: Numbering like this is something that would be checked in an accessibility audit; there are ways to make SVG numbering accessible though.